index.html

```diff
@@ -2,26 +2,81 @@
-     <h2> Experience: </h2>
-    <ul>
-     <li> "TJX", USA </li>
-     <li>"Miru Mir" educational center, Uzbekistan </li> 
-     <li>"Miru Mir" educational center, Uzbekistan </li> 
-     <li>"Hansung", automobile factory, South Korea </li>
-     <li>"Variant Optima", advertising agency, Uzbekistan </li>
-     <li> Secondary school 33, Uzbekistan </li>
-     </ul>
-     <h2> Connect: </h2>
-     <ul>
-         <li> <a href = "https://github.com/Elena-Nam"> GitHub </a> </li>
-         <li> <a href = "mailto:ek28150@gmail.com"> Email me </a> </li>
-     </ul>   
+    <main>
+    <header>
+            <div class="container">
+                <nav>
+                <a href="#leave_a_message"><strong> Leave a Message</strong></a>
+                <a href="#connect"><strong> Connect </strong></a>
+                <a href="#projects"><strong> Projects </strong></a>
+                <a href="#skills"><strong> Skills </strong></a>
+                <a href="#experience"><strong> Experience </strong></a>
+                <a href="#about"><strong> About </strong></a>
+                <a class="active" href="#home"><strong> Home </strong></a> 
+                </nav>
+            </div> 
+    </header>
+
+        <h1> Elena Nam </h1>
+        
+            <div class ="ticker">
+                <marquee>
+                    <p><strong> If you can dream it, you can do it </strong></p>
+                </marquee>
+            </div>
+
+    <section id="about">
+        <h2><em> About: </em></h2> 
+        <p> I am a teacher and learner. I am currently developing my skills on web development.</p>
+    </section>
+
+    <section id="experience">
+        <h2><em> Experience: </em></h2>
+        <ul>
+         <li> "TJX", USA </li>
+         <li>"Miru Mir" educational center, Uzbekistan </li> 
+         <li>"Miru Mir" educational center, Uzbekistan </li> 
+         <li>"Hansung", automobile factory, South Korea </li>
+         <li>"Variant Optima", advertising agency, Uzbekistan </li>
+         <li> Secondary school 33, Uzbekistan </li>
+         </ul>
+    </section> 
+
+    <section id="skills">
+        <h2><em> Skills </em></h2>
+    </section>
+     
+    <section id="projects">
+        <h2><em> Projects </em></h2>
+    </section>
+
+    <section id="connect">
+        <h2><em> Connect: </em></h2>
+        <ul>
+                <i class="fa fa-github" style="font-size:24px"></i>
+                 <a href = "https://github.com/Elena-Nam" target="_blank"> <strong> GitHub </strong> </a><br>
+        
+                <i class="fa fa-envelope-o" style="font-size:20px"></i>
+                <a href = "mailto:ek28150@gmail.com" target="_blank"> <strong> Email me </strong> </a> 
```

Comment:
the `<li>` tags for the Experience list items are missing now, so the bullet points will not show up. That should be an easy fix though.